CONTRIBUTING.md

```diff
@@ -95,6 +95,9 @@ into the source repository.
   contribution. Just comment on the PR and ask.
   Old and stalled [PRs are sometimes archived][archived-pull-requests] with the
   "State: archived" label, maybe one of them is also about the same topic.
+  There also may be a solution someone has done but just not made a PR, if that
+  is the case first raise an issue or ask on the devel@riot-os.org mailing
+  list
```

Comment:
I think we might end up receiving >10 emails per day if everyone would ask there before working on a new feature. And I expect at least twice the number of projects are started than actually are completed. I would likely unsubscribe the email list if we start receiving that volume on a daily bases, as I just don't have the time required to read that many emails.

Also, if the effort before opening a PR becomes to big, we end up with less PRs and fewer people contributing.

I think if problems turn up with competing PRs and no one has bad intents, it will be mostly a communication issue. If communication works properly, people should soon figure out if it is best to collaborate (both PRs target similar use cases or the requirements of the use cases do not conflict with each other); or if it is one of the rather rare cases the requirements of the intended use case conflict enough to justify competing solutions. (Examples of this are: GNRC vs lwIP, gcoap vs nanocoap vs libcoap, ...)